sky/provision/azure/instance.py

```diff
@@ -313,27 +313,6 @@ def _create_vm(
     )
     # poller.result() will block on async operation until it's done.
     logger.info(f'Created VM {vm_poller.result().name}.')
```

Comment:
Can we have an additional line for waiting for the poller? It is mainly for readability, logging is mostly for non-stateful operations.